python/src/main/python/drivers/shader_job_uniforms_to_spirv_fuzz_facts.py

```diff
@@ -79,9 +83,9 @@ def main() -> None:
                 constantWord=[int_representation])
             fact_list.append(dict(constantUniformFact=fact_constant_uniform))
 
-    with open(os.path.splitext(args.shader_job)[0] + ".facts", "w") as f:
+    with open(args.output_file, "w") as f:
```

Comment:
You didn't really change this, but it is best to use `encoding='utf-8', errors='ignore'`, or just use `runspv.open_helper`

Reply:
Done (the latter)